example/stateful_provider.dart

```diff
@@ -25,7 +25,7 @@ class MyApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return StatefulProvider<Bloc>(
       valueBuilder: (_) => Bloc(),
-      onDispose: (_, value) => value.dipose(),
+      dispose: (_, value) => value.dipose(),
```

Comment:
We would need to update the README after this as well.

Reply:
Right. There's a lot of documentation update needed

I'll fix tests first though, are most needs a rewrite.

Reply:
I'll fix that in another PR related to documentation